### PR TITLE
feat(core): doc database properties

### PR DESCRIPTION
**Pull request by pengx17**
fix AF-1454

1. move inline tags editor to components
2. add progress component
3. adjust doc properties styles for desktop
4. subscribe bs database links and display in doc info
5. move update/create dates to doc info
6. a trivial e2e test

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/T2klNLEk0wxLh4NRDzhk/eed266c1-fdac-4f0e-baa9-4aa00d14a2e8.mp4">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/eed266c1-fdac-4f0e-baa9-4aa00d14a2e8.mp4">10月23日.mp4</video>

### CODE REVIEW

**Comment by pengx17** [comment]
* **#8520** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8520?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/docs-search/services/docs-search.ts [hunk 1]
```diff
@@ -509,6 +509,134 @@ export class DocsSearchService extends Service {
...
+            : node.fields.docId[0],
+        rowId:
+          typeof node.fields.blockId === 'string'
+            ? node.fields.blockId
+            : node.fields.blockId[0],
+        databaseBlockId:
+          typeof node.fields.parentBlockId === 'string'
+            ? node.fields.parentBlockId
+            : node.fields.parentBlockId[0],
+        databaseName: JSON.parse(additional).databaseName as string,
```
Comment (EYHN) [inline packages/frontend/core/src/modules/docs-search/services/docs-search.ts]: additional may be undefined

File: packages/frontend/core/src/modules/doc-info/services/doc-database-backlinks.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+  // backlinks (docid:blockid:databaseBlockId)
+  //   -> related db rows (DatabaseRow[])
+  watchDbBacklinkRows$(docId: string) {
+    return this.docsSearchService.watchDatabasesTo(docId).pipe(
+      distinctUntilChanged(equalComparator),
+      switchMap(backlinks => {
+        return combineLatest(
+          backlinks.map(backlink => {
+            return this.watchDatabaseRow$(backlink);
```
Comment (EYHN) [inline packages/frontend/core/src/modules/doc-info/services/doc-database-backlinks.ts]: watchDbBacklinkRows will cause open doc to put pressure on storage. Can it be delayed until the user clicks on the collapsed area of ​​the database?

Comment (pengx17) [reply packages/frontend/core/src/modules/doc-info/services/doc-database-backlinks.ts]: yes, it is already implemented like you said. Data loading will only happen when the cell is being rendered (which is collapsed in doc info and collpased in database properties section by default)

**Review by EYHN — APPROVED** [review]

**Pull request merged**